Mathlib/Topology/Separation.lean

```diff
@@ -52,6 +55,12 @@ This file defines the predicate `SeparatedNhds`, and common separation axioms
 Note that `mathlib` adopts the modern convention that `m ≤ n` if and only if `T_m → T_n`, but
 occasionally the literature swaps definitions for e.g. T₃ and regular.
 
+### TODOs
+
+* Add perfectly normal and T6 spaces.
+* Use `has_separating_covers_iff_separated_nhds` to prove that perfectly normal spaces
```

Comment:
`has_separating_covers_iff_separated_nhds` is not following the name convention. It should be `hasSeparatingCovers_iff_separated_nhds`. The point is that underscores separate the different Lean declarations that are involved. This makes it much easier to see that this lemma is about `HasSeparatingCovers`.

Reply:
bc8a5bf